.github/workflows/mock-e2e-pr.yml

```diff
@@ -0,0 +1,71 @@
+name: Test e2e for PR
+
+on:
+  pull_request:
+
+jobs:
+  check-changed-files:
+    runs-on: ubuntu-20.04
+    name: Check changed files
+    outputs:
+      CANT_PASS_WITHOUT_TESTS: ${{ steps.decision.outputs.CANT_PASS_WITHOUT_TESTS }}
+    steps:
+      - id: files
+        uses: jitterbit/get-changed-files@b17fbb00bdc0c0f63fcf166580804b4d2cdc2a42
```

Comment:
This is coming from Akamai. Let's use the one from CloudFront because this library is not receiving any updates for a long time.

Reply:
This commit fixes it: https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/pull/7/commits/6758dcc7654b2e0fcbc9b46e4bf5b33b86c1e728